docs/try-it-out/longer-introduction.md

```diff
@@ -93,14 +93,16 @@ Add the If node:
 		If you didn't follow the step in the previous section to run the NASA node, you won't see any data to work with in this step.
 		///
 
-    2. Change the comparison operation to **Contains**.
+    2. Change the comparison operation to **String>Contains**.
```

Comment:
```suggestion
    2. Change the comparison operation to **String > Contains**.
```
Maybe? I think this is the format we've used elsewhere (but it's likely to be inconsistent).